Remotion/Web/Core/Res/Themes/NovaGray/HTML/TabbedMultiView.css

```
@@ -20,22 +20,27 @@ div.tabbedMultiView > div.wrapper
   right: 0;
   width: auto;
   height: auto;
+
+  display: grid;
+  grid-template-columns: auto;
+  grid-template-rows: max-content max-content auto max-content;
+  grid-template-areas: "topControls" "tabStrip" "view" "bottomControls";
 }
 
 div.tabbedMultiViewActiveView
 {
-  height: 80%;
-  width: inherit;
+  grid-area: view;
+  display: flex;
+  flex-direction: column;
+  overflow: hidden;
 }
 
 div.tabbedMultiViewActiveView > div.contentBorder
 {
-  position: relative;
   background-color: white;
-  margin: 0; 
+  margin: 1px; /* this small margin is needed to correctly display the outline in FF */
```

```suggestion
  margin: 0px;
```
Tested this, behavior is identical to before the change so no bugfix is needed.